app/services/france_connect_v2_open_id_client/callback.rb

```diff
@@ -1,5 +1,7 @@
 # voir https://github.com/france-connect/Documentation-AgentConnect/blob/main/doc_fs/technique_fca/endpoints.md
 module FranceConnectV2OpenIdClient
+  ALGORITHM = "ES256".freeze
```

Comment:
On pourrait être un tout petit peu plus explicite comme ça ?
```suggestion
  ES256_ALGORITHM = "ES256".freeze
```

Reply:
En effet c'est mieux !